### ISSUE TITLE
Map out UX work

### ISSUE THREAD

**Issue by bmw** [certbot/certbot#8247]
The next major project we're planning on working on is largely rewriting Certbot's output based on what we found during user testing. There is a design document for this at https://docs.google.com/document/d/1aKnQYEzCrZgYX-iuE-ErOdIKERUuLmZSX_BDb7d0M6I/edit?usp=sharing. I think we should essentially turn individual tasks in that design document into issues to allow people to easily see who is doing what, what work still needs to be done, what is blocked on what, etc. I think each issue should:

1. Be in the [UX project](https://github.com/certbot/certbot/projects/4) to allow us to easily find all of these issues.
2. Have a priority label.
3. If the work is blocked on another issue, mention that in the issue description and give the issue the "blocked" label.
4. Be in a milestone. I'd suggest either just using our current milestone or n+1 if the issue is blocked on more stuff than you think we can accomplish this month.
5. Describe the work in enough detail in the issue that anyone on the Certbot team could in theory take it.

**Comment by ohemorange** [comment]
This should now be complete, as all of the suggested output changes and outlined PRs have been turned into issues. There is some higher-level discussion that might be a good basis for future ideas, but otherwise, the concrete proposals are covered.

**Comment by bmw** [comment]
Thanks a lot for doing this!
